spec.html

```diff
@@ -22373,11 +22389,12 @@ <h1>Runtime Semantics: GetModuleNamespace ( _module_ )</h1>
           1. Assert: _module_.[[Status]] is not `"uninstantiated"`.
           1. Let _namespace_ be _module_.[[Namespace]].
           1. If _namespace_ is *undefined*, then
-            1. Let _exportedNames_ be ? _module_.GetExportedNames(&laquo; &raquo;).
+            1. Let _exportedNames_ be ? _module_.GetExportedNames(&laquo; &raquo;, _module_).
             1. Let _unambiguousNames_ be a new empty List.
-            1. For each _name_ that is an element of _exportedNames_, do
-              1. Let _resolution_ be ? _module_.ResolveExport(_name_, &laquo; &raquo;).
-              1. If _resolution_ is a ResolvedBinding Record, append _name_ to _unambiguousNames_.
+            1. If _exportedNames_ is not *null*, then
+              1. For each _name_ that is an element of _exportedNames_, do
```

Comment:
this already exists in master, but it's a somewhat confusing way of talking about iterating through a List to me. Typically I'd expect `For each _name_ of _exportedNames_, do`

Reply:
Sure I've updated this.